### DISCUSSION TITLE
clangd arguments reference

### DISCUSSION THREAD

**Discussion by laurelkeys** [clangd/clangd#2110] [Q&A] [answered]
Hi 👋

I failed to find documentation for all the available command line arguments that clangd may take (in particular, for VS Code's `"clangd.arguments"` array).

Is there some doc page (such as [Clang](https://clang.llvm.org/docs/ClangCommandLineReference.html)'s) I might have missed? If not, could you point me to the file(s) that do the parsing/declaration of arguments?
I presume that's not in this repo, as searching for e.g. `-function-arg-placeholders` or `-background-index` I couldn't find anything.

Thank you!

**Comment by HighCommander4** [comment, marked as answer]
You can run `clangd --help` on the command line to get a list of command line arguments and their descriptions.
